### PR TITLE
Feature/assert no page not found

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: test/performance/README.md [hunk 1]
```diff
@@ -4,22 +4,21 @@ The `profiles.jmx` [JMeter](http://jmeter.apache.org/) test will call the Profil
...
-| `throughput`                     | target throughput in samples per minute                            | 120                |
-| `csvfile`                        | csv data file of choices ID to use, either `ids_100.csv`, `ids_1200.csv`, or `ids_all.csv` (9000+), allowing a number of different GP pages to be visited during the test | ids_100.csv         |
+| Parameter                          | Description                                                                                                                                                               | Default                     |
+| :--------------------------------- | :-------------------------------------------------------------------                                                                                                      | -----------------------     |
+| `hostname`                         | URL of server to test                                                                                                                                                     | staging.beta.nhschoices.net |
```
Comment (markysoft) [inline test/performance/README.md]: Good spot

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles succeeded (http://profiles-pr-194.dev.beta.nhschoices.net)

File: test/performance/profiles.jmx [hunk 2]
```diff
@@ -101,7 +101,53 @@
           <stringProp name="HTTPSampler.response_timeout"></stringProp>
           <stringProp name="TestPlan.comments">Detected the start of a redirect chain</stringProp>
         </HTTPSamplerProxy>
-        <hashTree/>
+        <hashTree>
+          <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
+            <collectionProp name="Asserion.test_strings">
+              <stringProp name="1235244996">Page not found</stringProp>
```
Comment (markysoft) [inline test/performance/profiles.jmx]: Could the test assert against the response code, rather than the response text?

**Review by markysoft — CHANGES_REQUESTED** [review]

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/profiles succeeded (http://profiles-pr-194.dev.beta.nhschoices.net)

File: test/performance/profiles.jmx (hunk 2 above)
Comment (st3v3nhunt) [reply test/performance/profiles.jmx]: @markysoft I've done some more testing and I don't think there needs to be an explicit check for the 404 as this is done by default. I thought the exit code was being set to non-zero when there is an assertion check but this isn't the case so it makes no difference. Hopefully TeamCity can pick up on the number of tests that have failed from the report.

**Comment by c2s-dev** [comment]
:rocket: deployment of nhsuk/profiles succeeded (http://profiles-pr-194.dev.beta.nhschoices.net)

**Comment by st3v3nhunt** [comment]
No longer required

**Pull request closed without merging**